src/components/cluster/new/CreateRegularCluster.js

```diff
@@ -331,9 +340,51 @@ class CreateRegularCluster extends React.Component {
     });
   };
 
+  produceRAMAndCoresAWS = () => {
+    const { awsInstanceTypes } = this.state;
+    const instanceType = this.state.aws.instanceType.value;
+
+    // Check whether this.state.awsInstanceTypes is populated and that instance name
+    // in input matches an instance in the array
+    const instanceTypesKeys = Object.keys(awsInstanceTypes);
+
+    const hasInstances =
+      instanceTypesKeys.length > 0 &&
+      instanceTypesKeys.find(type => type === instanceType);
```

Comment:
`includes` is what we want here, is more readable also, but I am not removing the first part of the conditional because if array is undefined it will throw an error. But no need to use lenght with `include`.
Good catch!